modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -88,9 +88,9 @@ Note the worker with the role `worker-rt` that contains the string `4.18.0-211.r
 NAME                               	STATUS   ROLES           	AGE 	VERSION                  	INTERNAL-IP
 EXTERNAL-IP   OS-IMAGE                                       	KERNEL-VERSION
 CONTAINER-RUNTIME
-cnf-worker-0.example.com	          Ready	 worker,worker-rt   5d17h   v1.19.0-rc.2+aaf4ce1-dirty
+cnf-worker-0.example.com	          Ready	 worker,worker-rt   5d17h   v1.20.0-rc.2+aaf4ce1-dirty
 128.66.135.107   <none>    	        Red Hat Enterprise Linux CoreOS 46.82.202008252340-0 (Ootpa)
-4.18.0-211.rt5.23.el8.x86_64   cri-o://1.19.0-90.rhaos4.6.git4a0ac05.el8-rc.1
+4.18.0-211.rt5.23.el8.x86_64   cri-o://1.20.0-90.rhaos4.6.git4a0ac05.el8-rc.1
```

Comment:
@StephenJamesSmith Just a heads-up on this update. Please confirm that this change is okay. Thanks!

Reply:
Delete **-rc.2+aaf4ce1-dirty**.

Reply:
Thanks @StephenJamesSmith, I deleted that. Look good now?